app/test_main.py

```diff
@@ -1,3 +1,57 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestAllCases:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, result", [
+            (
+                0,
+                0,
+                [0, 0]
+            ),
```

Comment:
Improve the readability by placing arguments on one line
```suggestion
            (0, 0, [0, 0]),
```